fury/decorators.py

```diff
@@ -43,3 +46,95 @@ def doctest_skip_parser(func):
         new_lines.append(code)
     func.__doc__ = "\n".join(new_lines)
     return func
+
+
+def keyword_only(func):
```

Comment:
Can you rename `warn_on_args_to_kwargs`

also, this function needs parameter like `from_version` and `until_version`

Reply:
@skoudoro

All right, thanks for your feedback.

Reply:
reminder here